pos_order_to_sale_order/README.rst

```diff
@@ -15,74 +15,87 @@ creation from the Point of Sale.
 In the POS UI, buttons has been added to create a sale order and discard
 the current POS order.
 
-This module is usefull in many cases, for exemple :
+This module is useful in many cases, for example :
 
-* take orders with a very simple interface
 
+* take quotations with a very simple interface
+* take orders with a very simple interface
+* allow clients to pay later
 * if you have some customers that come every day in your shop, but want to
-  have a unique invoice at the end of the month. With that module, you can
-  create a sale order and deliver products every time to keep your stock value
-  correct, and to create a unique invoice, when you want.
+  have a unique invoice at the end of the month. With this module, you can
+  create a sale order and deliver products every time he comes 
+  to keep your stock value correct and to create a unique invoice periodically.
 
 
-Three options are available:
+Four options are available:
```

Comment:
This could be moved to configuration section

Reply:
It's more about what this module do rather than where/how to configure it.